Makefile

```diff
@@ -109,6 +109,7 @@ install_cli_cpu_neural_compressor:
 
 install_cli_cuda_pytorch:
 	pip install -e .[testing,timm,diffusers,peft,autoawq,auto-gptq,bitsandbytes,deepspeed]
+	curl -s https://raw.githubusercontent.com/huggingface/optimum-benchmark/main/scripts/install_quantization_libs.py | python - --install-autoawq-from-source --install-autogptq-from-source
```

Comment:
where's the script in this PR I don't see it ? and why do you need to curl if the repo is cloned locally ?

Reply:
Sorry, I forgot to add it to my push... 😅 Also the script will only be available on main when this is merged. Right now you can test the command with `https://raw.githubusercontent.com/huggingface/optimum-benchmark/23bf7da73541bc114d4869c7c09aa7acf34025d5/scripts/install_quantization_libs.py`

The advantage is that not everyone install optimum-benchmark from source. So people with pip install optimum-benchmark can directly use our script without cloning the whole repo

Reply:
> Also the script will only be available on main when this is merged

yes but it should be pushed to the OR so that I can review it 😅
> The advantage is that not everyone install optimum-benchmark from source. So people with pip install optimum-benchmark can directly use our script without cloning the whole repo
Yes but quant libs installation from source is not a "feature" of `optimum-benchmark` it's a script that we're gonna use for our own specific needs (llm-perf). if a user wants to use the script then they could either follow the installation instructions in the quant libs directly, or clone `optimum-benchmark` and use the make commands, or curl the file on their own, let's not over complicate this as it's a very super user feature (we're only responsible for the quant kernels slowdown in #214  when it impacts our benchmarks, otherwise it's on the quant lib)

Reply:
Ok then I will just remove the curl and adjust the documentation

Reply:
Done. Let me know what you think